<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="help_info_button">
        <property name="text">
         <string>‚ùìÂú∫ÊôØËØ¥Êòé</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="demo_button">
        <property name="text">
         <string>üì∞ÊºîÁ§∫</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Â§öË°®ÂåπÈÖçÂú∫ÊôØ</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="release_info_button">
        <property name="text">
         <string>üí°Âú∫ÊôØÊõ¥Êñ∞‰ø°ÊÅØ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_button">
        <property name="text">
         <string>ÈáçÁΩÆÊâÄÊúâ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Á¨¨‰∏ÄÊ≠• ÔºöÊãñÊãΩÊàñÁÇπÂáª‰∏ä‰º†Excel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="step1_help_info_button">
        <property name="text">
         <string>‚ùìÁ¨¨‰∏ÄÊ≠•ÁöÑÂ∏ÆÂä©‰ø°ÊÅØ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="add_main_table_button">
            <property name="text">
             <string>‚ûï</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>‰∏ªË°®</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="main_tables_table">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="autoScroll">
           <bool>true</bool>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Ë°®Âêç</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ÈÄâ‰∏≠Â∑•‰ΩúË°®</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ê†áÈ¢òÊâÄÂú®Ë°å</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Êìç‰ΩúÊåâÈíÆ</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>__Ë°®Ë∑ØÂæÑ</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="add_help_table_button">
            <property name="text">
             <string>‚ûï</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>ËæÖÂä©Ë°®</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="help_tables_table">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="autoScroll">
           <bool>true</bool>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Ë°®Âêç</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ÈÄâ‰∏≠Â∑•‰ΩúË°®</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ê†áÈ¢òÊâÄÂú®Ë°å</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Êìç‰ΩúÊåâÈíÆ</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>__Ë°®Ë∑ØÂæÑ</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="add_condition_button">
        <property name="text">
         <string>‚ûï</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Á¨¨‰∫åÊ≠• ÔºöËÆæÁΩÆÂåπÈÖçÊù°‰ª∂‰∏éÊìç‰Ωú</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="step2_help_info_button">
        <property name="text">
         <string>‚ùìÁ¨¨‰∫åÊ≠•ÁöÑÂ∏ÆÂä©‰ø°ÊÅØ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="conditions_table">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="columnCount">
       <number>9</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>‰∏ªË°®ÂåπÈÖçÂàó</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ËæÖÂä©Ë°®Âêç</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ËæÖÂä©Ë°®ÂåπÈÖçÂàó</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ÂåπÈÖçÂøΩÁï•ÂÜÖÂÆπ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ÂàóÔºö‰ªéËæÖÂä©Ë°®Â¢ûÂä†</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ÂàóÔºöÂåπÈÖçÈôÑÂä†‰ø°ÊÅØÔºàÊñáÂ≠óÔºâÂèØÁºñËæë</string>
       </property>
       <property name="background">
        <color>
         <red>207</red>
         <green>240</green>
         <blue>198</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ÂàóÔºöÂåπÈÖçÈôÑÂä†‰ø°ÊÅØÔºàË°åÊï∞Ôºâ</string>
       </property>
       <property name="background">
        <color>
         <red>207</red>
         <green>240</green>
         <blue>196</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Âà†Èô§Êª°Ë∂≥Êù°‰ª∂ÁöÑË°å</string>
       </property>
       <property name="background">
        <color>
         <red>242</red>
         <green>195</green>
         <blue>189</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Êìç‰ΩúÊåâÈíÆ</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="run_button">
        <property name="text">
         <string>‚ñ∂</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Á¨¨‰∏âÊ≠• ÔºöÊâßË°å‰∏é‰∏ãËΩΩ</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="step3_help_info_button">
        <property name="text">
         <string>‚ùìÁ¨¨‰∏âÊ≠•ÁöÑÂ∏ÆÂä©‰ø°ÊÅØ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="result_detail_text">
        <property name="text">
         <string>ÂÖ±ÂåπÈÖçÔºö--Ë°åÔºà--%Ôºâ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="result_detail_info_button">
        <property name="text">
         <string>üí°ÂêÑËæÖÂä©Ë°®ÁªüËÆ°</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="view_result_button">
        <property name="text">
         <string>ÊîæÂ§ßÁªìÊûúÊü•Áúãüîç</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="include_detail_checkbox">
        <property name="text">
         <string>‰∏ãËΩΩÂåÖÊã¨ÂåπÈÖçÈôÑÂä†‰ø°ÊÅØ</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="download_result_button">
        <property name="text">
         <string>‰∏ãËΩΩÁªìÊûúüìÑ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="result_table"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
